### PR TITLE
docs(homepage): provide English documents of style part

### PR DESCRIPTION
**Pull request by chiyu1996**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated/passed for the code you will submit.
- [x] Documentation has added or updated.
- [ ] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters.
- [ ] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=1855) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=1855) before we can accept your contribution.<br/><hr/>**stanzyzhang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=1855) it.</sub>

File: docs/en-us/style/appearance.md [hunk 1]
```diff
@@ -1,290 +1,291 @@
-# 外观
+# Appearance
 
-包含了前景、背景、边框、透明度、字体等外观样式
+Includes foreground,background,border,opacity,font and other appearance styles
 
 # borderColor
 
-| 类型               | 必需 | 支持平台
+| Type               | Required| Support platform
```
Comment (zoomchan-cxj) [inline docs/en-us/style/appearance.md:7]: Support platform -> Supported Platforms

Comment (chiyu1996) [reply docs/en-us/style/appearance.md]: > Updated

File: docs/en-us/style/appearance.md [hunk 2]
```diff
@@ -1,290 +1,291 @@
...
+[Hippy-React example](https://github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/src/components/Text/index.jsx)
 
-[Hippy-Vue 范例](https://github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-p.vue)
+[Hippy-Vue example](https://github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-p.vue)
 
-文本的修饰线颜色
+Decoration line color for text
 
-| 类型   | 必需 | 平台 |
+| Type   | Required| Platform|
```
Comment (zoomchan-cxj) [inline docs/en-us/style/appearance.md:173]: Platform ->  Supported Platforms  中文我们后面修改下

File: docs/en-us/style/appearance.md [hunk 3]
```diff
@@ -1,290 +1,291 @@
...
 | ------ | -------- | --------|
-| object: { x: number, y: number }  | 否 | Android、iOS |
+| object: { x: number, y: number }| No|  Android,iOS |
 
 # textShadowOffsetX
 
-> * 最低支持版本 2.10.0
-> * 注意 hippy-vue class 样式只支持合并写法 `text-shadow-offset: 1px 1px`，不支持拆分
+* Minimum supported version 2.10.0
+* Note that the Hippy-Vue class style only supports the combined writing method 'text shadow offset: 1px 1px', and does not support splitting
```
Comment (zoomchan-cxj) [inline docs/en-us/style/appearance.md:232]: writing method -> writing format

File: docs/en-us/style/appearance.md [hunk 4]
```diff
@@ -1,290 +1,291 @@
...
 
-[Hippy-React 范例](https://github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/src/components/Image/index.jsx)
+[Hippy-React example](https://github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/src/components/Image/index.jsx)
+
+[Hippy-Vue example](https://github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-img.vue)
 
-[Hippy-Vue 范例](https://github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-img.vue)
 
-对图片进行染色(对非纯色图片进行有透明度的染色时，Android 和 iOS 的 `blendMode` 默认值有差异)
+Tint  the image (When tinting the non-solid color image with transparency, there is a difference in the default value of 'blendmode' between Android and iOS)
```
Comment (zoomchan-cxj) [inline docs/en-us/style/appearance.md:280]: Tint  the  之间多了空行

File: docs/en-us/style/layout.md [hunk 5]
```diff
@@ -1,431 +1,434 @@
-# 布局
+# Layout
 
-Hippy 的样式排版使用了 Flex 布局。值得注意的是，尚不兼容网页的百分比布局。
+Hippy's style layout uses a Flex layout. It is worth noting that the `PercentFrameLayout` of web pages is not yet compatible.
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:3]: Hippy's style layout uses a Flex  layout ->  Hippy's style layout uses `Flex` 就可以了

File: docs/en-us/style/layout.md [hunk 6]
```diff
@@ -1,431 +1,434 @@
...
+[[MDN Docs]](//developer.mozilla.org/en-US/docs/Web/CSS/align-items)
 
-`alignItems`决定了子元素在次轴方向的排列方式（此样式设置在父元素上）。例如若子元素本来是沿着竖直方向排列的（即主轴竖直，次轴水平），则 `alignItems` 决定了它们在水平方向的排列方式。此样式和CSS中的`alignItems`表现一致，默认值为 `stretch`。
+`alignItems` determines how the child elements are arranged in the direction of the secondary axis (this style is set on the parent element). For example, if the child elements are originally arranged in a vertical direction (i.e., The primary axis is vertical and the secondary axis is horizontal), the `alignItems` determines how they are arranged in the horizontal direction. Its behavior is consistent with that on CSS `align-items`(default is`stretch`).
 
-| 类型                                                            | 必需 |
+| Type                                                            | Required|
 | --------------------------------------------------------------- | -------- |
-| enum('flex-start', 'flex-end', 'center', 'stretch', 'baseline') | 否       |
+| enum('flex-start', 'flex-end', 'center', 'stretch', 'baseline') |false       |
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:17]: false ->  No

File: docs/en-us/style/layout.md [hunk 7]
```diff
@@ -1,431 +1,434 @@
...
+[[MDN Docs]](//developer.mozilla.org/en-US/docs/Web/CSS/align-self)
 
-`alignSelf`决定了元素在父元素的次轴方向的排列方式（此样式设置在子元素上），其值会覆盖父元素的`alignItems`的值。其表现和 CSS 上的`align-self`一致（默认值为`auto`）。
+`alignSelf` determines how elements are arranged in the direction of the secondary axis of the parent element (this style is set on the child element), and its value overrides the value of the parent element `alignItems`. Its behavior is consistent with that on CSS `align-self`(default is`auto`).
 
-| 类型                                                                    | 必需 |
+| Type                                                                    | Required|
 | ----------------------------------------------------------------------- | -------- |
-| enum('auto', 'flex-start', 'flex-end', 'center', 'stretch', 'baseline') | 否       |
+| enum('auto', 'flex-start', 'flex-end', 'center', 'stretch', 'baseline') |false       |
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:27]: false -> No

File: docs/en-us/style/layout.md [hunk 8]
```diff
@@ -1,431 +1,434 @@
...
+`backgroundImage` value can be directly passed into the background image address or gradient.
 
-| 类型            | 必需 |
+| Type            | Required|
 | --------------- | -------- |
-| string | 否      |
+| string |false      |
 
-> `2.8.1` 版本后支持终端本地图片能力，可通过 webpack `file-loader` 加载。
+>after`2.8.1` the version, it supports the local image capability of the terminal and can be loaded through webpack `file-loader`.
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:40]: after`2.8.1` the version  -> after the version `2.8.1`

Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:40]: terminal -> native

File: docs/en-us/style/layout.md [hunk 9]
```diff
@@ -1,431 +1,434 @@
...
-> 渐变色目前支持 `linear-gradient`  线性渐变 `(最低支持版本 2.8.0）` [[MDN 文档]](//developer.mozilla.org/zh-CN/docs/orphaned/Web/CSS/linear-gradient())，支持使用 `linear-gradient([ [ [ <angle> | to [top | bottom] || [left | right] ],]? <color-stop>[, <color-stop>]+)` 格式; 其中 `angle` 支持 deg、turn、rad 单位；`color-stop` 支持设置多个颜色和百分比。DEMO： [HippyReact](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/src/components/View/index.jsx) [HippyVue](//github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-div.vue)
+>Gradient currently support `linear-gradient` linear gradient`(Minimum supported version 2.8.0）` [[MDN Docs]](//developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient),and supports the use of `linear-gradient([ [ [ <angle> | to [top | bottom] || [left | right] ],]? <color-stop>[, <color-stop>]+)` format. Among them, `angle` supports `deg`, `turn` and `rad` units, and `color-stop` supports setting multiple colors and percentages. DEMO:[Hippy-React](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/src/components/View/index.jsx) [Hippy-Vue](//github.com/Tencent/Hippy/blob/master/examples/hippy-vue-demo/src/components/demos/demo-div.vue)
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:42]: `linear-gradient` linear gradient`  去掉后面的

File: docs/en-us/style/layout.md [hunk 10]
```diff
@@ -1,431 +1,434 @@
...
 > <br/>
 > <br/>
-> 注意：
+>Note:
 >
-> + Android 如果使用 `to [top | bottom] || [left | right]` 四个顶角设置渐变角度，`color-stop` 不能使用百分比；
-> + iOS `color-stop` 百分比只能从小到大依次显式设置，不能部分省略，即 `red 10%, yellow 20%, blue 50%`，不能是 `red 10%, yellow 20%, blue 10%`
+>+ Android can not use percentages if you use`to [top | bottom] || [left | right]` four top corners to set the gradient angle `color-stop`;
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:47]: 改成： Android can not use percentages for `color-stop` if you use `to [top | bottom] || [left | right]` four top corners

File: docs/en-us/style/layout.md [hunk 11]
```diff
@@ -1,431 +1,434 @@
...
+[[MDN Docs]](//developer.mozilla.org/en-US/docs/Web/CSS/background-position)
 
-`backgroundPositionX` 指定背景图片的初始位置的横轴X坐标。
+`backgroundPositionX` specifies the horizontal X-coordinate of the initial position of the background image.
 
-| 类型            | 必需 |
+| Type            | Required|
 | --------------- | -------- |
-| number | 否       |
+| number |false       |
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:58]: false ->  No

File: docs/en-us/style/layout.md [hunk 12]
```diff
@@ -1,431 +1,434 @@
...
 # display
 
-Hippy 默认采用 Flex 布局。同时，因为仅支持 Flex 布局，所以不需要手写 display: flex 即可使用。
+Hippy adopts Flex layout by default. Also, because it only supports Flex layout, it can be used without handwritten `display: flex`.
 
-| 类型            | 必需 |
+| Type            | Required|
 | --------------- | -------- |
-| enum('flex') | 否       |
+| enum('flex') |false       |
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:94]: 改成 No

File: docs/en-us/style/layout.md [hunk 13]
```diff
@@ -1,431 +1,434 @@
...
 # flexGrow
 
-[[MDN 文档]](//developer.mozilla.org/zh-CN/docs/Web/CSS/flex-grow)
+[[MDN Docs]](//developer.mozilla.org/en-US/docs/Web/CSS/flex-grow)
 
-`flexGrow` 定义伸缩项目的扩展能力。它接受一个不带单位的值做为一个比例。主要用来决定伸缩容器剩余空间按比例应扩展多少空间。
+`flexGrow` defines the extensibility of the scaling project. It accepts a value without units as a scale. It is mainly used to determine how much space the remaining space of the telescopic container should be expanded proportionally.
 
-如果所有伸缩项目的 `flex-grow` 设置了 `1`，那么每个伸缩项目将设置为一个大小相等的剩余空间。如果你给其中一个伸缩项目设置了 `flex-grow` 值为 `2`，那么这个伸缩项目所占的剩余空间是其他伸缩项目所占剩余空间的两倍。
+If all scaling items are`flex-grow` set,`1` then each scaling item will be set to an equal amount of remaining space. If you set a`flex-grow` value of for`2` one of the scaling items, the remaining space for that scaling item is twice as large as the remaining space for the other scaling items.
```
Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:133]: set,`1`  多了逗号

Comment (zoomchan-cxj) [inline docs/en-us/style/layout.md:133]: value of for`2` 是什么意思

File: docs/en-us/structure/introduction.md [hunk 14]
```diff
@@ -1,22 +1,22 @@
-# 介绍
+# Introduction
 
-# Hippy 2.x 架构
+# Hippy 2.x architecture
 
-Hippy 2.x 架构主要分成三层，UI(JS) 层 `Hippy-React` 和 `Hippy-Vue` 负责驱动 UI 指令生成；中间层 [C++ HippyCore](structure/core.md) 负责抹平平台差异性和提供高性能模块；渲染层 `Android` 和 `iOS` 负责提供终端底层模块、组件，并与布局引擎通信
+The Hippy 2.x architecture is mainly divide into three layers: UI(JS) layer `Hippy-React` and`Hippy-Vue` instruction generation responsible for driving UI; The middle layer [C++ HippyCore](structure/core.md) is responsible for smoothing platform differences and providing high-performance modules; Render Layers `Android` and `iOS` are responsible for providing the bottom-modules and components of the terminal and communicating with the layout engine.
```
Comment (zoomchan-cxj) [inline docs/en-us/structure/introduction.md:5]: terminal -> native

File: docs/en-us/guide/exception.md [hunk 15]
```diff
@@ -1,14 +1,14 @@
-# 异常捕获
+# Exception capture
 
-Hippy 对于 JS 代码里没有处理的异常错误，可以通过监听 `uncaughtException` 事件进行捕获。
+Hippy can catch unexpected errors that are not handled in JS code by listening `uncaughtException` for events.
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/exception.md:3]: `uncaughtException`  for events  -> 去掉 for

File: docs/en-us/guide/exception.md [hunk 16]
```diff
@@ -1,14 +1,14 @@
...
 
-!> 当前 `uncaughtException` 暂时无法捕获 `Promise` 里的错误
+!> currently `uncaughtException` unable to catch `Promise` errors in
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/exception.md:5]: unable to catch `Promise` errors in 改成  unable to catch errors in  `Promise`

File: docs/en-us/guide/integration.md [hunk 17]
```diff
@@ -1,121 +1,121 @@
-# 开始接入
+# Getting Started
 
-Hippy 采用 `monorepo` 进行代码管理，版本前端和终端统一，前端可以直接引入对应的 NPM 包，终端可通过发布分支源码接入或通过对应的包管理仓库引入。
+Hippy uses `monorepo` for code management to unify the versions of the front-end and native. The front-end can directly introduce the corresponding NPM package, and the native can access it through publishing branch source code or through the corresponding package management Warehouse.
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:3]: Warehouse -> repository

File: docs/en-us/guide/integration.md [hunk 18]
```diff
@@ -1,121 +1,121 @@
...
 
-Hippy 已经提供了完整的[前端和终端范例](//github.com/Tencent/Hippy/tree/master/examples)，可直接基于我们现有的范例开始 App 开发。若想快速体验 Hippy，可按照 [README 步骤](https://github.com/Tencent/Hippy/blob/master/README.zh_CN.md#-%E5%BC%80%E5%A7%8B) 将 DEMO 运行起来 。 如果要在已有的 App 里整合 Hippy，请继续阅读下面的`终端集成`章节。
+Hippy already provides a complete [front-end and native paradigm](//github.com/Tencent/Hippy/tree/master/examples) to start App development directly based on our existing paradigm. For a quick experience of Hippy, follow [the README steps](https://github.com/Tencent/Hippy/blob/master/README.zh_CN.md#-%E5%BC%80%E5%A7%8B) to run DEMO. If you want to integrate Hippy into an existing App, continue to read the following `native access` chapter.
 
-# 终端接入
+# native access
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:7]: native 首字母大写

File: docs/en-us/guide/integration.md [hunk 19]
```diff
@@ -1,121 +1,121 @@
...
 
-如果要接入 Hippy 到现有终端项目，请参考 [Android集成](android/integration.md) 和 [iOS集成](ios/integration.md) 教程。
+If you want to access Hippy to an existing native project, refer to the [Android integration](android/integration.md) and [iOS integration](ios/integration.md) tutorials.
 
-# 前端接入
+# front-end access
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:11]: 首字母大写

File: docs/en-us/guide/integration.md [hunk 20]
```diff
@@ -1,121 +1,121 @@
...
-Hippy 同时支持 React 和 Vue 两种 UI 框架，通过 [@hippy/react](//www.npmjs.com/package/@hippy/react) 和 [@hippy/vue](//www.npmjs.com/package/@hippy/vue) 两个包提供实现。
+Hippy supports both React and Vue UI frameworks, which are implemented through the [@hippy/react](//www.npmjs.com/package/@hippy/react) and [@hippy/vue](//www.npmjs.com/package/@hippy/vue) packages.
 
 ## hippy-react
 
-[[hippy-react 介绍]](hippy-react/introduction.md) [[范例工程]](//github.com/Tencent/Hippy/tree/master/examples/hippy-react-demo)
+[[hippy-react introduces]](hippy-react/introduction.md) [[example project]](//github.com/Tencent/Hippy/tree/master/examples/hippy-react-demo)
 
-hippy-react 工程暂时只能通过手工配置初始化，建议直接 clone 范例工程并基于它进行修改。
+The hippy-react project can only be initialized by manual configuration for the time being. It is recommended to directly clone the instance project and modify it based on it.
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:19]: instance project  -> demo project

File: docs/en-us/guide/integration.md [hunk 21]
```diff
@@ -1,121 +1,121 @@
...
-| @hippy/hippy-live-reload-polyfill   | live-reload 必备脚本 - 会在调试模式编译时注入代码到工程里 |
-| @hippy/hippy-dynamic-import-plugin  | 动态加载插件 - 拆分出子包用于按需加载
-| @babel/plugin-x                     | Babel 其余相关插件，如 `@babel/plugin-proposal-nullish-coalescing-operator` 等 |
-| case-sensitive-paths-webpack-plugin | Webpack 插件，对 import 文件进行大小写检查 |
-| file-loader                         | 静态文件加载                               |
-| url-loader                          | 静态文件以 Base64 形式加载                 |
+| Package name                                | Description                                                                                            |
+| ----------------------------------- |--------------------------------------------------------------------------------------------------------|
+| @hippy/hippy-live-reload-polyfill   | live-reload prerequisite script - it will inject code into the project when compiling in debugging mode |
+| @hippy/hippy-dynamic-import-plugin  | Dynamically load plugin - split sub packages for on-demand loading
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:55]: Dynamically load -> dynamic load

File: docs/en-us/guide/integration.md [hunk 22]
```diff
@@ -1,121 +1,121 @@
...
 
-线上包和开发调试用包主要有两个区别：
+There are two main differences between online packages and development and debugging packages:
 
-1. 开启了 production 模式，去掉调试信息，关闭了 `watch`（watch 模式下会监听文件变动并重新打包）。
-2. 终端内很可能不止运行一个 Hippy 业务，所以将共享的部分单独拆出来做成了 `vendor` 包，这样可以有效减少业务包体积，这里使用了 [DllPlugin](//webpack.js.org/plugins/dll-plugin/) 和 [DllReferencePlugin](//webpack.js.org/plugins/dll-plugin/#dllreferenceplugin) 来实现。需要说明的是生成的 `vendor` 包正常情况下是不需要特别更新的，但是如果更新了也要注意一下向上兼容性，不要因为分包导致业务崩溃。
+1. Turn on production mode, remove debug information, and turn off `watch` (watch mode listens for file changes and repackages).
+2. More than one Hippy service is likely to run in the native, so the shared part is separated into a `vendor` package, which can effectively reduce the volume of the service package. [DllPlugin](//webpack.js.org/plugins/dll-plugin/) and [DllReferencePlugin](//webpack.js.org/plugins/dll-plugin/#dllreferenceplugin) are used to realize this. It should be noted that the generated `vendor` package does not need to be specially updated under normal circumstances, but if it is updated, pay attention to upward compatibility, and do not cause service crash due to subcontracting.
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:78]: service ->  business

File: docs/en-us/guide/integration.md [hunk 23]
```diff
@@ -1,121 +1,121 @@
...
-| [hippy-webpack.ios-vendor.js](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/scripts/hippy-webpack.ios-vendor.js) | iOS Vendor 包配置             |
-| [hippy-webpack.android.js](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/scripts/hippy-webpack.android.js) | Android 业务包配置            |
-| [hippy-webpack.android-vendor.js](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/scripts/hippy-webpack.android-vendor.js) | Android Vendor 包配置             |
+| Configuration file                                                     | Description                                                 |
+| ------------------------------------------------------------ |-------------------------------------------------------------|
+| [vendor.js](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/scripts/vendor.js) | Shared parts that need to be included in the vendor package |
+| [hippy-webpack.ios.js](//github.com/Tencent/Hippy/blob/master/examples/hippy-react-demo/scripts/hippy-webpack.ios.js) | iOS service package configuration                           |
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:83]: service -> business

File: docs/en-us/guide/integration.md [hunk 24]
```diff
@@ -1,121 +1,121 @@
...
 ```js
 import { Hippy } from '@hippy/react';
 import App from './app';
 
 new Hippy({
-  appName: 'Demo',  // 终端分配的业务名称
-  entryPage: App,   // 对应业务启动时的组件
-  silent: false,    // 设置为 true 可以关闭框架日志输出
+  appName: 'Demo',  // Service name assigned by native
+  entryPage: App,   //  Corresponding to the component when the service is started
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/integration.md:102]: service -> business

File: docs/en-us/guide/network-request.md [hunk 25]
```diff
@@ -67,79 +67,73 @@ async function getMoviesFromApi() {
...
 ### readyState
 
-`number，只读属性`
-> 当前WebSocket的状态，取值为以下4种：
-> * `0` - WebSocket正在连接中（Connecting）；
-> * `1` - WebSocket连接建立成功，当前可以收发数据（Open）；
-> * `2` - WebSocket连接正在关闭（Closing）；
-> * `3` - WebSocket连接已关闭（Closed）；
+`number, read-only attribute`
+>The current WebSocket status has the following 4 values:
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/network-request.md:102]: > 和  >*  后面与文字加空格

File: docs/en-us/guide/network-request.md [hunk 26]
```diff
@@ -67,79 +67,73 @@ async function getMoviesFromApi() {
...
+>*`0` WebSocket is connecting(Connecting).
+>*`1` Websocket connection is established successfully. Data can be sent and received at present(Open).
+>*`2` Websocket connection closing(Closing).
+>*`3` Websocket connection closed(Closed).
 
 ### url
 
-`string，只读属性`
-当前 WebSocket 使用的 url
+`string, read-only attribute`url currently used by WebSocket
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/network-request.md:110]: `string, read-only attribute`url currently used by WebSocket   分开两行

File: docs/en-us/guide/network-request.md [hunk 27]
```diff
@@ -67,79 +67,73 @@ async function getMoviesFromApi() {
...
 
 ### onopen
 
-`function，只写属性`
-设置当前 WebSocket 在连接建立成功后的回调函数
+`function, write-only attribute`Set the callback function of the current WebSocket after the connection is successfully established
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/network-request.md:114]: `function, write-only attribute`Set the callback function of the current WebSocket after the connection is successfully established  分开两行

File: docs/en-us/guide/network-request.md [hunk 28]
```diff
@@ -192,8 +186,8 @@ export default class WebSocketExpo extends React.Component {
 
 # Cookie
 
-Hippy 在接收来自服务器的 `set-cookie` header 时，会自动种入 cookie，下次再请求同域名服务时，就自动带上之前种下的 cookie。
+When Hippy receives the `set-cookie` header from the server, it will automatically seed the cookie, and the next time it requests the same domain name service, it will automatically bring the cookie planted before.
 
-但和浏览器不同，Hippy 内提供提供了 NetworkModule 提供了对 cookie 读取和修改，详情可以参考 hippy-react 的 [NetworkModule](hippy-react/modules.md?id=networkmodule)，或者 hippy-vue 的 [Vue.Native.Cookie](/hippy-vue/vue-native.md?id=cookie) 文档。
+However, unlike that browser, the `NetworkModule` provided in Hippy provide reading and modification of cookies. For details, please refer to the [NetworkModule](hippy-react/modules.md?id=networkmodule) document of Hippy-React or the [Vue.Native.Cookie] (/hippy-vue/vue-native.md?id=cookie) document of Hippy-Vue.
```
Comment (zoomchan-cxj) [inline docs/en-us/guide/network-request.md:191]: Hippy 内提供提供了 NetworkModule 提供了对 cookie 读取和修改  这句中文本身有问题，中英都要修改一下

File: docs/en-us/structure/introduction.md [hunk 29]
```diff
@@ -1,22 +1,22 @@
...
+Hippy is upgrading the 3.x architecture. In 3.x, the specific implementation in the business and rendering layers can be switched according to the actual user scenario: the business layer is no longer limited to JS driver, and other languages (such as DSL/Dart/WASM, etc.) can also be selected for driving. DOM Manager sinks from Java/OC to C++, as an intermediate hub, in addition to receiving and processing the messages from the upper layer to create and maintain DOM Tree, it is also responsible for interfacing and communicating with different rendering engines, typesetting engines and debugging tools. In the rendering layer, the rendering engine can also select other rendering renderers, such as Flutter(Voltron) rendering, in addition to supporting the existing Native rendering. Hippy 3.x can make up for some shortcomings of current Hippy 2.x in performance, double-end consistency, and component support, so look forward to it!
```
Comment (zoomchan-cxj) [inline docs/en-us/structure/introduction.md:17]: the business layer  改成 the driver layer 吧

**Pull request merged**